src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/FeatureSwitches.kt

```diff
@@ -27,12 +27,6 @@ class FeatureSwitches(private val environment: Environment) {
   }
 }
 
-@Component
-class Toggles(private val featureSwitches: FeatureSwitches) {
-  fun isMasterPublicAndPrivateNotes() = featureSwitches.isEnabled(Feature.FEATURE_MASTER_PUBLIC_PRIVATE_NOTES)
-}
-
```

Comment:
Should we leave the component in, even if with just a test/placeholder toggle? There will be others coming.

Reply:
Maybe.  Don't mind either way.